<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(23, 27, 33);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_main">
     <widget class="QLabel" name="main_msg_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>811</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AppleSDGothicNeoB00</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
      </property>
      <property name="text">
       <string>메뉴를 선택하세요</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>776</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="move_registration_page_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>등록</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="move_enter_page_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>등교</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="move_outside_page_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>외출</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="move_return_page_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>복귀</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="move_leave_page_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>하교</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_outside">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>776</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="eating_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>식사</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="gs25_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>도서관</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="etc_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>기타</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="outside_msg_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>811</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AppleSDGothicNeoB00</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
      </property>
      <property name="text">
       <string>사유를 선택하세요</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_last_member">
     <widget class="QLabel" name="last_msg_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>811</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AppleSDGothicNeoB00</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
      </property>
      <property name="text">
       <string>마지막 퇴실자인가요?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>776</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="yes_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>네</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="no_button">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
border-radius: 15px;</string>
         </property>
         <property name="text">
          <string>아니요</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_fingerprint">
     <widget class="QLabel" name="fingerprint_msg_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>811</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AppleSDGothicNeoB00</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
      </property>
      <property name="text">
       <string>지문을 입력하세요</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_registration">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>231</y>
        <width>811</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="3">
        <widget class="QPushButton" name="three_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="six_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="zero_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="four_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="two_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="one_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="seven_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="five_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="backspace_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>←</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="eight_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="nine_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="enter_button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AppleSDGothicNeoB00</family>
           <pointsize>30</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>↲</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="registration_msg_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>811</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AppleSDGothicNeoB00</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;
QLineEdit {color: white;}
QLineEdit:placeholder {color: white;}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>학번을 입력하세요</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="header_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>60</height>
     </rect>
    </property>
    <widget class="QLabel" name="current_time_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>470</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AppleSDGothicNeoB00</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>현재 시각 : </string>
     </property>
    </widget>
    <widget class="QLabel" name="current_member_label">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>0</y>
       <width>241</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AppleSDGothicNeoB00</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>현재 인원 : 명</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_main_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AppleSDGothicNeoB00</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30, 34, 40);
color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>[←]</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
